pe32me162ir_pub.ino

```diff
@@ -73,6 +73,12 @@ static const int PIN_IR_TX = 10; // digital pin 10
 const int PULSE_THRESHOLD = 100;  // analog value between 0 and 1023
 #endif //OPTIONAL_LIGHT_SENSOR
 
+#define MQTT_TLS
+/* Use TLS with pinned certificate for secure MQTT.
+ * Put the 20-byte fingerprint of the server certificate in config.h. */
+#define MQTT_AUTH
+/* Use password authentication (only valid when MQTT_TLS is set) */
```

Comment:
These should be moved to config.h so as not to disturb those who aren't using these.